### PR TITLE
Add joining the RSE to the new joiner checklist

### PR DESCRIPTION
**Pull request by llewelld**
## Summary
Joining the Society of Research Software Engineering is mentioned on the "Systems Set Up" page for new joiners, but isn't currently featured in the new joiner Checklist. This change adds it in.

## Related issues
N/A

## Screenshots
N/A

---

### Updates
N/A

### CODE REVIEW

**Comment by llewelld** [comment]
Just a small change, but I'm suggesting to add it as personally I find the checklist to be a particularly useful page.

**Comment by DavidBeavan** [comment]
Does this relate to #52, cos it looks like it might close that, at least partially

**Comment by llewelld** [comment]
Sorry, I hadn't noticed #52. I'm afraid this change is much smaller in scope (e.g. I didn't add anything about RSE benefits or how to claim back). I'd be happy to try to expand this change to try, but as it stands I'm afraid I don't think this makes much of a dent in #52.

**Comment by llewelld** [comment]
Rebased.

**Comment by DavidBeavan** [comment]
> I'm afraid this change is much smaller in scope

No worries, continue your changes as was, but if you fancy having a crack at the larger issue get stuck in

**Comment by JimMadge** [comment]
@llewelld @DavidBeavan How much work would it be to solve #52? Is it worth doing that in this PR?

**Comment by DavidBeavan** [comment]
I don't think it should take long, and I've just added more detail to the issue, so it just needs expanding upon. My priorities are elsewhere this month, but it would help to have the info out there if yous have time

**Comment by JimMadge** [comment]
> I don't think it should take long, and I've just added more detail to the issue, so it just needs expanding upon. My priorities are elsewhere this month, but it would help to have the info out there if yous have time

:+1:

That's great 🙌 . It looks like there is enough detail in the issue that it can just be translated to markdown/book format.

@llewelld Are you interested in adding that here too, or shall we do that in another PR?

**Comment by llewelld** [comment]
Thanks for the additional info.

I'd be happy to have a go at adding the material to this PR. I should be able to push an updated version today.

**Comment by llewelld** [comment]
I tried adding the info to the "Systems Set Up" page, but it started to grow, and ultimately felt like the wrong place, so I ended up creating a new page for it. It's still in the onboarding section, but maybe there's a better place for it?

I've just pushed these changes as an additional commit on top.

**Comment by llewelld** [comment]
Closes #52.

**Review by JimMadge — COMMENTED** [review]
LGTM :+1:

Not sure if @DavidBeavan wants to have a preview.

My only note is I'm not sure if the society uses 'SocRSE'. I suppose *we* can decide to do that in our own book but they might not like the styling.

File: content/docs/onboarding/new_joiners/checklist.md [hunk 1]
```diff
@@ -81,3 +81,4 @@ If anything is unclear, please figure it out by e.g. asking your buddies, and th
 ## Ongoing activity sign-up
 
 - [ ] "Tech Talks" (Tuesdays 12:30pm), link on: https://github.com/alan-turing-institute/DataScienceSkills/wiki/Lunchtime-Tech-Talks
+- [ ] Read up about and optionally join the [Society of Research Software Engineering](https://society-rse.org/), including the [Slack workspace](https://ukrse.slack.com/join/signup)
```
Comment (DavidBeavan) [inline content/docs/onboarding/new_joiners/checklist.md:84]: I'd suggest a stronger stance on joining, given the benefits and conference discount. Suggest 'optionally' -> 'you are recommended to'

**Comment by DavidBeavan** [comment]
> I'm not sure if the society uses 'SocRSE'

Yeah, that's fine, it is used by SocRSE, not a prob.

**Comment by DavidBeavan** [comment]
@llewelld BTW I really appreciate you taking on the extra work to cover the issue

**Comment by andrewphilipsmith** [comment]
Looks good to me.

Good to see #52 completed too.

**Review by andrewphilipsmith — APPROVED** [review]

File: content/docs/onboarding/new_joiners/checklist.md (hunk 1 above)
Comment (llewelld) [reply content/docs/onboarding/new_joiners/checklist.md]: Thanks for the suggestion; I've made this change.

**Comment by llewelld** [comment]
> @llewelld BTW I really appreciate you taking on the extra work to cover the issue

No problem, it's minor effort, but good to have in the handbook. Thanks for providing the info.

**Comment by llewelld** [comment]
Rebased.

**Comment by JimMadge** [comment]
@all-contributors please add @DavidBeavan for ideas, review

**Pull request merged**